public/scripts/timeline.js

```diff
@@ -1597,6 +1608,22 @@ function isValidDate(dateString)
     return day > 0 && day <= monthLength[month];
 };
 
+function generateMementoURIList(object)
+{
+    var URIM = [];
+
+    $.each(object,function(index,obj){
+        if($(obj.event_html).attr("src").indexOf("notcaptured") < 0){
+            URIM[index] = obj.event_link;

```

Comment:
If the `if` condition here is to filter some entries off, then the `URIM` array will be a sparse one in which skipped entries will be empty at their corresponding index. As a result, in the final output, there will be some empty lines. You can use `Array.push()` method here to eliminate empty entries in the first place like this:

```suggestion
            URIM.push(obj.event_link);
```